File: api/src/main/java/com/fiera/api/dtos/UserDTO.java
package com.fiera.api.dtos;

import javax.persistence.Id;
import javax.persistence.GeneratedValue;
import javax.validation.constraints.NotNull;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class UserDTO {

    @Id 
    @GeneratedValue
    private Integer UserId;
    @NotNull
    private String DocNumber;
    @NotNull
    private String FirstName;
    @NotNull
    private String LastName;
    private String Email;
    private String Phone;
    private String Address;
}

File: NetCore/TestWebAPIWindowsContainers/TestWebAPIWindowsContainers/DTO/UserDTO.cs
namespace TestWebAPIWindowsContainers.DTO
{
    /// <summary>
    /// Class to expose data
    /// </summary>
    public class UserDTO
    {
        public int UserId { get; set; }
        public string DocNumber { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
    }
}
